app/anti_virus_check.py

```diff
@@ -0,0 +1,139 @@
+import collections
+import json
+import os
+import time
+
+import requests
+from sdc.rabbit.exceptions import QuarantinableError, RetryableError, BadMessageError
+
+from app import create_and_wrap_logger
+from app import settings
+
+logger = create_and_wrap_logger(__name__)
+
+AVResult = collections.namedtuple('AVResult', 'safe ready scan_results')
+
+
+class AntiVirusCheck:
+    def __init__(self, tx_id):
+        self.bound_logger = logger.bind(tx_id=tx_id)
+
+    def send_for_av_scan(self, payload):
+        self.bound_logger.debug("A/V scanned enabled {}".format(settings.ANTI_VIRUS_ENABLED))
```

Comment:
Is this debug line useful? The method is only called if `settings.ANTI_VIRUS_ENABLED` is `True` and the info log line below would only be output if it is enabled.